tests/fixtures/xcs/build.atpkg

```diff
@@ -3,25 +3,25 @@
 
      :tasks {
         :build-lib {
-            :tool "atllbuild"
-            :source ["src/**.swift"]
-            :outputType "static-library"
+            :tool "llbuild"
+            :sources ["src/**.swift"]
+            :output-type "static-library"
             :name "Foo"
-            :compileOptions ["-enable-testing"]
+            :compile-options ["-enable-testing"]
         }
         :build-tests {
-            :tool "atllbuild"
-            :source ["test/**.swift"]
-            :outputType "executable"
+            :tool "llbuild"
+            :sources ["test/**.swift"]
+            :output-type "executable"
             :name "footests"
             :dependencies ["build-lib"]
             :xctestify true
-            :xctestStrict true
-            :linkWithProduct["Foo.a"]
+            :xctest-strict true
```

Comment:
We need to update the documentation at https://github.com/AnarchyTools/atbuild/blob/master/docs/xctest.md